<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="boxr2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>690</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bleu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rouge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jaune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxr1">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>470</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rouge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bleu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jaune</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxc1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Rouge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bleu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jaune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxr">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>240</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Vert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rouge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bleu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jaune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxc2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>470</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bleu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rouge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jaune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxt1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>690</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jaune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rouge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bleu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="boutonCompo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Composer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonQuitter">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonArreter">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>650</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>ArrÃªter</string>
    </property>
   </widget>
   <widget class="QSlider" name="boutonVitesse">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>710</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonDeplacerVertical">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>650</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Deplacer Vertical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonDeplacerHorizontal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>650</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Deplacer Horizontal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonCancelAll">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>CancelAll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonValider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonCancel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>710</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxTc1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>240</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x1.5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxTr">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>240</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x1.5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxTc2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>470</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x1.5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxTr1">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>470</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x1.5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxTt1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>690</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x1.5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boxTr2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>690</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x1.5</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
